components/Pages.php

```diff
@@ -746,15 +753,15 @@ public function edit_page_form() {
 		if ( $has_precode ) {
 			pods_deprecated( 'Pod Page precode has been deprecated, please use WP Page Templates or hook into the pods_content filter instead of embedding PHP.', '2.1' );
 
-			if ( PODS_DISABLE_EVAL ) {
+			if ( defined( 'PODS_DISABLE_EVAL' ) && ! PODS_DISABLE_EVAL ) {
```

Comment:
Same as above: 
Is this change correct? I still see an else statement that basically returns that same-ish message.
Maybe just remove the if/else statement and just return a single message?